lib/msal-common/package.json

```diff
@@ -47,7 +47,7 @@
         "test:coverage": "nyc --reporter=text mocha --exit",
         "test:coverage:only": "npm run clean:coverage && npm run test:coverage",
         "build:modules": "rollup -c",
-        "build:modules:watch": "rollup -cw",
+        "build:modules:watch": "npm run lint && rollup -cw",
```

Comment:
I would like to have the build tasks just perform the build step, and not perform other tasks.

Reply:
That way if I just want to build and not worry about lint, I can.

Reply:
I don't know why this was added to the watch command, but build:modules and build:modules:watch will let you build without linting, but I would like a command to run during the pipeline that also does lint checks.

Reply:
I think lint should be added as a separate task (which I started doing for #1700).